### PR TITLE
Add test skeleton for test_setting_guaranteed_engine_cpu 

### PR DESCRIPTION
**Pull request by yasker**
https://github.com/longhorn/longhorn/issues/1318

Also remove test skeleton for test_replica_rebuild_concurrent_limit.
https://github.com/longhorn/longhorn/issues/307

### CODE REVIEW

File: manager/integration/tests/test_settings.py [hunk 1]
```diff
@@ -198,3 +198,29 @@ def check_tolerations_set(current_toleration_list, set_tolerations):
...
 
     return current_tolerations == set_tolerations
+
+
+@pytest.skip(reason="TODO")
+def test_setting_guaranteed_engine_cpu():
+    """
+    Test setting Guaranteed Engine CPU
+
+    1. Change the setting to `0.1`
```
Comment (sowmyav27) [inline manager/integration/tests/test_settings.py:208]: Does a change in the "Guaranteed Engine CPU" setting value, result in instance manager pods to be recreated ?

File: manager/integration/tests/test_settings.py [hunk 2]
```diff
@@ -198,3 +198,29 @@ def check_tolerations_set(current_toleration_list, set_tolerations):
...
+    2. Wait for all the instance manager pods to recreated and become running
+    3. Verify every pod has guaranteed CPU set correctly to 100m
+    4. Change the setting to `0`
+    5. Wait for all the instance manager pods to recreated and become running
+    6. Verify every pod no longer has guaranteed CPU set
+    7. Change the setting to `200m`
```
Comment (sowmyav27) [inline manager/integration/tests/test_settings.py:214]: Is this an invalid value check ?

Comment (yasker) [reply manager/integration/tests/test_settings.py:214]: `200m` is a valid value. It's the same as `0.2`. See https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/

Comment (yasker) [reply manager/integration/tests/test_settings.py:214]: Thinking about it, yes, we should have an invalid value check.

File: manager/integration/tests/test_settings.py (hunk 1 above)
Comment (yasker) [reply manager/integration/tests/test_settings.py:208]: Yes.

File: manager/integration/tests/test_settings.py [hunk 3]
```diff
@@ -198,3 +198,29 @@ def check_tolerations_set(current_toleration_list, set_tolerations):
...
+    8. Wait for all the instance manager pods to recreated and become running
+    9. Verify every pod no longer has guaranteed CPU set to 200m
```
Comment (sowmyav27) [inline manager/integration/tests/test_settings.py:216]: if 200m is same as 0.2, why is it expected here that we "no longer have guaranteed CPU set to 200m" ?

Comment (yasker) [reply manager/integration/tests/test_settings.py:216]: Good catch. Typo.

**Pull request merged**